packages/docs/src/components/GithubLinks.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Button } from '@cmsgov/design-system';
+import classnames from 'classnames';
+
+interface GithubLinks {
+  /**
+   * Describes if the links are on a dark background
+   */
+  onDark?: boolean;
+}
+
+/**
+ * GithubLinks - a reusable component for Github links for downloading code & the Github repo
+ */
+const GithubLinks = ({ onDark }: GithubLinks) => {
+  return (
+    <>
```

Comment:
Might be nice to have the github logo in here? I guess the icon isn't a part of our set maybe not :shrug: Just always something I look for instinctively.

Reply:
I agree. Right now, I'm porting over the exact button from the current site, but I think there have been (multiple) requests to use the GH logo when we redesign!